pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: unittest
-version: 0.11.7
+version: 0.11.8
 author: Dart Team <misc@dartlang.org>
 description: A library for writing dart unit tests.
 homepage: https://github.com/dart-lang/old_unittest
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=2.0.0-dev.61.0 <3.0.0'
```

Comment:
nit: this should be `<2.0.0` still, we aren't bumping packages to `<3.0.0` until Dart 2 stable is released

Reply:
Not absolutely sure, but it seems to be running for the use-cases where we are using it.

We should stop using `unittest` in the SDK completely, as soon as we find the time to fix it.